pipeline/01_parser.py

```
@@ -174,19 +653,90 @@ def parse_file(file_path):
+            with open(output_path, encoding="utf-8") as f:
+                prev = json.load(f)
+            for n in prev:
+                try:
+                    wr = n["url"].split("wr_id=")[1].split("&")[0]
+                except (KeyError, IndexError):
+                    continue
+                existing[wr] = {
+                    a["name"]: a.get("parsed_text", "")
+                    for a in n.get("attachments", []) if a.get("name")
+                }
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)